content/doc/book/updating/_pre-upgrade-wizard.adoc

```diff
@@ -0,0 +1,18 @@
+////
+This file is only meant to be included as a snippet in other documents.
+There is a version of this file for the general 'Updating Jenkins' page (index.adoc).
+This file is for the index.adoc page used in the general 'Updating Jenkins' page.
+////
+
+[[pre-upgrade-wizard]]
+
+== Before you upgrade
+
+. Verify the test environment is functioning correctly after it is upgraded.
+
+. Ensure you have created a backup.
+
+. Verify all running builds are complete.
+
+. Java 11 is required for your Jenkins instance, the operations center, Jenkins controllers, and all agents for Jenkins LTS 2.303.3.
```

Comment:
```suggestion
. Java 11 is required for a modern Jenkins instance, the operations center, Jenkins controllers, and all agents for Jenkins as of the 2.357 weekly and Long Term Support release of 2.361.1.
```
[Java 11 is required](https://www.jenkins.io/blog/2022/06/28/require-java-11/#:~:text=Beginning%20with%20Jenkins%202.357%20(released,)%2C%20Jenkins%20supports%20Java%2017.) when updating to Jenkins LTS 2.361.1 (we are now on 2.375.1) or weekly (starting with 2.357, and we are now on 2.386)
Suggesting to make sure the current requirements are clear and visible